posts/2025-05-04-ask/index.qmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "Ask and you shall receive"
+description: |
+  Discussion of evolving complexity and value of the Bioconductor ecosystem. 
+author:
+  - name: Vince Carey
+    url: https://vjcitn.github.io/
+date: 2025-05-06
+creative_commons: CC BY
+categories:
+  - education
+image: "crazy.jpg"
+---
+title: "ask2025"
+author: "Vince Carey"
+---
+
+# In the beginning 
+
+... there was a 9-track tape shipped from
+Bell Labs.  S would be installed for "unix".  Extensions
+could be programmed in Fortran.  S REPORT presaged
+Sweave and knitr: troff, eqn, pic could be
+interlarded with S computations.  I have lost the associated
+technical report and the "Extending the S system" book.
+
+# Fast forward:
+
+![Bioconductor in 2024](crazy.jpg)
```

Comment:
You might want to include "crazy" in the caption, so that readers don't spend too long trying to understand the drawing! 😅